Meta/lint-clang-format.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd "${script_path}/.." || exit 1
+
+if ! command -v clang-format-10 >/dev/null 2>&1 ; then
+    echo "clang-format-10 is not available. Either skip this script, or install clang-format-10."
+    exit 1
+fi
```

Comment:
Maybe we can extend the message to include a hint for people running this script on non-Ubuntu systems - like Fedora:

```
$ command -v clang-format-10; echo $?
1
$ command -v clang-format; echo $?
0
$ clang-format --version
clang-format version 10.0.0 (Fedora 10.0.0-2.fc32)
```

I think creating a symlink on those systems will make the script usable.

Reply:
It's either `clang-format-10` or `clang-format`, why not just try both :shrug:

Reply:
Now it tries both, and even warns when using a non-10 version of `clang-format` :)